css/style.css

```diff
@@ -349,3 +355,61 @@ body {
   align-items: center;
   color: #253858;
 }
+
+.footer {
+  background-color: #6070ff;
+  font-style: normal;
+  font-family: Poppins, sans-serif;
+  height: 779px;
+  border-radius: 100px 0 0 0;
+  padding-top: 103px;
+  padding-right: 30px;
+  padding-left: 30px;
+}
+
+.contactme {
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+.form {
+  flex-wrap: wrap;
+}
+
+.form-description {
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebff;
+  text-align: center;
+  margin-bottom: 60px;
+}
+
+.form-element {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 17px;
+  line-height: 20px;
+  color: #172b4d;
+  flex: 1;
+  padding: 0 24px 0;
+  margin-bottom: 20px;
+  border-radius: 8px;
+  border: 1px solid #cfd8dc;
+}
+
+input.form-element {
+  height: 48px;
+  width: 327px;
+}
+
+textarea.form-element {
+  height: 170px;
+}
+
```

Comment:
- [ ] As the previous reviewer requested. Kindly add all the buttons interactions such as hover, active.  as shown in the figma template.
![image](https://user-images.githubusercontent.com/43698511/130141317-e472db14-dea0-4c33-855c-e939034e197c.png)

Reply:
Kindly follow the requested requirements and provide the correct colors and styles.